test/sql/tdigest.sql

```diff
@@ -952,3 +953,11 @@ FROM (
         unnest(percentile_cont(ARRAY[0.01, 0.99]) WITHIN GROUP (ORDER BY x)) AS b
     FROM data
 ) foo;
+
+-- verify <value, count> API
+select sum(tdigest_count(t)), round(tdigest_percentile(t, 0.5)::numeric, 2)
+ from (
+ select tdigest(v*w, w, 100) as t
+ from (
+ select unnest(v) as v, unnest(w) as w from (select ARRAY[0.1,0.2,0.3] as v,  ARRAY[50,50,100] as w) as q
+ ) as s) as s;
```

Comment:
This tests the `tdigest(value, count, compression)` aggregate.

Lets add a test that shows that `tdigest_percentile(value, count, compression, percentile)` gives the same output on the same input. 

And of course the others functions.

Reply:
added a missing test case. Still need to address the other functions.